src/commands/generateProject.ts

```diff
@@ -40,14 +39,13 @@ export async function generateProject(clickedFileUri: vscode.Uri | undefined): P
       return;
     }
 
-    const packageName = getPackageName(targetDirectory.fsPath);
+    const packageName = await prompts.askForPackageName(targetDirectory.fsPath);
```

Comment:
Add a note to the README that the user will be prompted for package name as an input